minecraft@.service

```diff
@@ -58,9 +58,9 @@ ExecStart=/bin/sh -c \
 ExecReload=/usr/bin/screen -p 0 -S mc-%i -X eval 'stuff "reload"\\015'
 
 ExecStop=/usr/bin/screen -p 0 -S mc-%i -X eval 'stuff "say SERVER SHUTTING DOWN. Saving map..."\\015'
+ExecStop=/bin/sleep 5
 ExecStop=/usr/bin/screen -p 0 -S mc-%i -X eval 'stuff "save-all"\\015'
 ExecStop=/usr/bin/screen -p 0 -S mc-%i -X eval 'stuff "stop"\\015'
-ExecStop=/bin/sleep 10
```

Comment:
This line is there because of a bug in older systemd versions. There it was reported to have failed if screen exited too quickly.
And also delays restarts of the service if the service fails, so please let this line present.